lansa-vmss-windows-autoscale-sql-database/createUiDefinition.json

```diff
@@ -188,7 +209,48 @@
                         },
                         "constraints": {
                             "required": true
-                        }
+                        },
+                        "visible": "[equals(steps('SecurityConfig').keyVaultEnabledForTemplateDeployment, 'false')]"
+                    },
+                    {
+                        "name": "keyVaultName",
+                        "type": "Microsoft.Common.TextBox",
+                        "label": "Key Vault Name",
+                        "toolTip": "The name of the keyvault that contains the secret.",
+                        "constraints": {
+                            "required": true,
+                            "regex": "^(?!\\s*$).+",
+                            "validationMessage": "The value must not be empty."
+                        },
+                        "visible": "[equals(steps('SecurityConfig').keyVaultEnabledForTemplateDeployment, 'true')]"
+                    },
+                    {
+                        "name": "keyVaultCertificateID",
+                        "type": "Microsoft.Common.TextBox",
+                        "label": "Key Vault Certificate ID",
+                        "toolTip": "Base 64 encoded String representing the SSL certificate",
+                        "constraints": {
+                            "required": true,
+                            "regex": "^(?!\\s*$).+",
+                            "validationMessage": "The value must not be empty."
+                        },
+                        "visible": "[equals(steps('SecurityConfig').keyVaultEnabledForTemplateDeployment, 'true')]"
+                    },
+                    {
+                        "name": "keyVaultSecretID",
```

Comment:
I'm pretty sure there is no Secret for a Key Vault Certificate. You have permission to use it, or you don't.

Reply:
Yes I agree with that, now SecretKey parameter is removed.